ksz8863/examples/two_ports_mode/pytest_two_ports_mode.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+# SPDX-License-Identifier: Apache-2.0
+import os
+import sys
+
+import pytest
+
+from pytest_embedded import Dut
+
+# Import test_common.py from parent directory
+sys.path.insert(1, '../../')
+from test_common import SwitchSSH
+from test_common import VirtualMachineSSH
+
+IP_ADDRESS_RUNNER = os.environ.get('IP_ADDRESS_RUNNER')
+IP_ADDRESS_ENDNODE = os.environ.get('IP_ADDRESS_ENDNODE')
+IP_ADDRESS_SWITCH = os.environ.get('IP_ADDRESS_SWITCH')
+SSH_VM_USERNAME = os.environ.get('SSH_VM_USERNAME')
+SSH_VM_PASSWORD = os.environ.get('SSH_VM_PASSWORD')
+SSH_SW_USERNAME = os.environ.get('SSH_SW_USERNAME')
+SSH_SW_PASSWORD = os.environ.get('SSH_SW_PASSWORD')
+SSH_PORT = 22
+
+runner = VirtualMachineSSH('Runner')
+endnode = VirtualMachineSSH('Endnode')
+switch = SwitchSSH('Switch')
+
+
+@pytest.fixture(scope='session', autouse=True)
+def prepare_vms_and_ksz8863():
+    # Connect to virtual machines
+    runner.connect(IP_ADDRESS_RUNNER, SSH_VM_USERNAME, SSH_VM_PASSWORD)
+    endnode.connect(IP_ADDRESS_ENDNODE, SSH_VM_USERNAME, SSH_VM_PASSWORD)
+    switch.connect(IP_ADDRESS_SWITCH, SSH_SW_USERNAME, SSH_SW_PASSWORD)
+
+
+def test_ksz8863_two_ports_link(dut: Dut) -> None:
+    # Wait for ESP32 to initialize
+    dut.expect('main_task: Returned from app_main()')
+    # Check that both ports will act as expected when link is brought up and down
+    switch.bring_port(3, 'down')
```

Comment:
Again, ports need to be configurable. I've already had such comment in `simple_switch`. Bogdan, please remember that this type of comment is meant for the whole PR for the next time.